mount/mount_setattr.go

```diff
@@ -0,0 +1,99 @@
+// +build linux
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package mount
+
+import (
+	"runtime"
+	"syscall"
+	"unsafe"
+)
+
+// There is no support of mount_setattr in golang
+// "golang.org/x/sys/unix" package and even in glibc
```

Comment:
Now we have https://pkg.go.dev/golang.org/x/sys@v0.0.0-20211205182925-97ca703d548d/unix#MountSetattr

Reply:
Thanks, will update this part.

Reply:
Rebased, updated according to recently introduced MountSetattr() support in "golang.org/x/sys/unix" package.